oefen_api/.vscode/launch.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Normaal gesproken laat je dit soort files (editor configuratie, locale configuratie files) uit je repository. Dit kun je doen door, in dit geval, de `.vscode` folder in `.gitignore` te zetten.